letsencrypt-auto-source/pieces/pipstrap.py

```diff
@@ -57,7 +57,7 @@ def check_output(*popenargs, **kwargs):
     from urllib.parse import urlparse  # 3.4
 
 
-__version__ = 2, 0, 0
+__version__ = 1, 5, 0
```

Comment:
It looks like this is reverting `pipstrap` slightly too far as I updated it to 1.5.1 in e0ae356aa35adf22d154113e06dd01409df93bba. You can find that version of pipstrap at https://github.com/erikrose/pipstrap/blob/1.5.1/pipstrap.py.

Reply:
Indeed. Fallback to 1.5.1.